app/inbox/migrations/0002_auto_20181220_1421.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 2.1.2 on 2018-12-20 14:21
```

Comment:
Can you just delete the existing migration at `inbox/migrations/0001_` and recreate the migrations?